MultiThreading.Task5.Threads.SharedCollection/Program.cs

```diff
@@ -2,24 +2,55 @@
  * 5. Write a program which creates two threads and a shared collection:
  * the first one should add 10 elements into the collection and the second should print all elements
  * in the collection after each adding.
- * Use Thread, ThreadPool or Task classes for thread creation and any kind of synchronization constructions.
+ * Use Thread, ThreadPool or Task classes for thread creation 
+ * and any kind of synchronization constructions.
  */
 using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace MultiThreading.Task5.Threads.SharedCollection
 {
     class Program
     {
+        static Dictionary<int, string> items = new Dictionary<int, string>();
         static void Main(string[] args)
         {
             Console.WriteLine("5. Write a program which creates two threads and a shared collection:");
-            Console.WriteLine("the first one should add 10 elements into the collection and the second should print all elements in the collection after each adding.");
-            Console.WriteLine("Use Thread, ThreadPool or Task classes for thread creation and any kind of synchronization constructions.");
+            Console.WriteLine("the first one should add 10 elements into the collection");
+            Console.WriteLine("and the second should print all elements in the collection after each adding.");
+            Console.WriteLine("Use Thread, ThreadPool or Task classes for thread creation");
+            Console.WriteLine("and any kind of synchronization constructions.");
             Console.WriteLine();
 
             // feel free to add your code
+            var task1 = Task.Factory.StartNew(() => addItem());
+            var task2 = Task.Factory.StartNew(() => printItem());
```

Comment:
The 2nd task should print all elements in the collection after each adding (in other words, if the collection contains elements from 1 to 10, the second thread should print something like [1], [1, 2], [1, 2, 3], …, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], the number of elements should increase).

Reply:
[Print items after each add](https://github.com/okutbay/01.multithreading.v.1.2/pull/1/commits/a39e837279060dd74606f7351936758791c3c563)